node/cli/src/export_metadata_cmd.rs

```diff
@@ -52,7 +52,7 @@ impl ExportMetadataCmd {
 		let maybe_hash = client.hash(block_number.into())?;
 		let block_hash = maybe_hash.ok_or_else(|| Error::from("Block not found"))?;
 
-		let metadata: Bytes = api.metadata(block_hash).unwrap().into();
+		let metadata: Bytes = api.metadata_at_version(block_hash, 15u32).unwrap().unwrap().into();
```

Comment:
metadata_at_version returns an option, thus the second unwrap after unwrapping the result

Reply:
I’m not sure if I’m overlooking something, but it seems like metadata_at_version only accepts one argument. Could you please confirm?
https://github.com/frequency-chain/frequency/blob/9fd47cf90b23b3030749790aceffbc726bd3dd40/runtime/frequency/src/apis.rs#L88

Reply:
Ahh its wrapped around `impl_runtime_apis`.